tools/to_ipynb.py

```diff
@@ -0,0 +1,116 @@
+import os
```

Comment:
Reorder imports and simplify multiple imports:

import os
import re
import nbformat
from nbformat.v4 import new_code_cell
from nbformat.v4 import new_markdown_cell
from nbformat.v4 import new_notebook
from redbaron import RedBaron
import yaml